qiskit/providers/ibmq/job/ibmqjob.py

```diff
@@ -290,6 +290,45 @@ def status(self) -> JobStatus:
 
         return self._status
 
+    def done(self) -> bool:
+        """Return whether the job is done.
```

Comment:
Can you use the same wording as `JobStatus`? Like "Return whether the job has successfully run".